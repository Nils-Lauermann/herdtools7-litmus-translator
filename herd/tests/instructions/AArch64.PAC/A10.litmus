AArch64 A10
Variant=pac,const-pac-field

{ 0:x0=x; int64_t 0:x1=42; int64_t x = 53 }

P0            ;
  pacda x0,x1 ;
  pacdzb x0   ;
  autda x0,x1 ;
  autdzb x0   ;
  ldr x1,[x0] ;
forall
  ( 0:x1=53 )
