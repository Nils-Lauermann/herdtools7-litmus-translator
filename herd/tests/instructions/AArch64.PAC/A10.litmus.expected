Test A10 Required
States 1
0:X1=53;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X1=53)
Observation A10 Always 1 0
Hash=180175b47aae92c49d0142601e283543

